spec/integration/workbook-xlsx-reader.spec.js

```diff
@@ -32,13 +32,13 @@ describe('WorkbookReader', () => {
               throw new Error('Promise unexpectedly fulfilled');
             },
             err => {
-              expect(err.message).to.equal('Max row count exceeded');
+              expect(err.message).to.equal('Max row count (10) exceeded');
             }
           );
       });
 
       it('should fail fast on a huge file', function() {
-        this.timeout(20000);
+        this.timeout(5000);
```

Comment:
This test is currently super flaky on `master`, this PR fixes it - and so confidently, that we can decrease the timeout from 20s to 5s